src/classes.py

```diff
@@ -2,21 +2,43 @@
 # constructor
 
 # YOUR CODE HERE
+
+class LatLon:
+    def __init__(self, lat, lon):
+        self.lat = lat
+        self.lon = lon
         
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
 # YOUR CODE HERE
+class Waypoint(LatLon):
+    def __init__(self, name, lat, lon):
```

Comment:
Cool! You could also modify lat and lon to be default arguments just in case no values are passed into the function

Reply:
That would probably be advisable thanks.